stories/typography.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import '../src/css/base.css';
+
+const typoDemoItemStyle = {
+  height: '80px',
+  borderBottom: 'solid gray 1px',
+  paddingTop: '10px',
+  paddingLeft: '10px',
+};
+
+storiesOf('General/Typography', module)
+  .add('Typography normal', () => (
+    <div>
+      <div className="typo-demo-block" style={typoDemoItemStyle}>
+        <h1 className="h1">
```

Comment:
I don't think we need a `h1` class, in our `base.css` we should just override the h1 element itself
Same with h2, h3, etc.